charts/puris-frontend/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
Please add missing license header, `sources` and `home` key, [TRG 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-2#the-chartyaml-file).

Reply:
Applied the changes as suggested!